### ISSUE TITLE
FP1 - ver104sym5fp1r1 - f909d5a9-d9a7-4318-9575-d59ee9dd5c73

### ISSUE THREAD

**Issue by m3d** [robotika/subt-artf#142]
Toto je extra beh se symetrii a pouze jednim robotem Freyja A2700R (levy by skoncil zaseknuty na kolejich, viz predesly beh #141 ). Robot dokazal ziskat **5 bodu!** tak si mozna zaslouzi i zaznam ;). Take to byl dlouhy beh, takze mne i zajima jak moc je co zalogovane a jestli s postupem casu nemame napr. redukovat logovani obrazu??

**Comment by zwn** [comment]
:1st_place_medal:

**Comment by jisa** [comment]
Pro me hodne necekane.

Docela by me zajimalo, jak je mozne, ze se pri navratu nezasekla na kolejich. Ale to se do logu s vysokou pravdepodobnosti neveslo :-/

**Comment by m3d** [comment]
No ja koukal na ten predesly beh a tam se od kolmych koleji "odrazila" a tim padem uz nelaborovala na "nebezpecnem uzemi" a dal asi koleje nebyly? Hlavne to resilo ty vertical shafts, kam nepadala. A konecne s tim flipem to i celkem rychle prozkoumavala :). Tj. ve FP1 je ted challenge ta leva strana s kolejema.

**Comment by m3d** [comment]
```
A2700R__breadcrumb___0: 29.69 -0.72 0.00
A2700R__breadcrumb___1: 32.19 -30.42 0.00
A2700R__breadcrumb___2: 31.96 -59.83 -2.49
A2700R__breadcrumb___3: 61.23 -0.31 0.00
A2700R__breadcrumb___4: 82.53 19.39 -5.00
A2700R__breadcrumb___5: 112.17 19.67 -10.00
min x: -3.50 y: -87.20
max x: 575.20 y: 108.20
width: 5826px height: 1994px
```
![state tlog](https://user-images.githubusercontent.com/5664353/114993565-3834c000-9e9c-11eb-9b1b-d62a2922215f.png)

**Comment by m3d** [comment]
Log konci v 1940s (sim)
![finals-p1-log-end](https://user-images.githubusercontent.com/5664353/114995258-ed1bac80-9e9d-11eb-8eb7-cccdcceefb0b.jpg)
(mozna jezdime tech 15min a T2000 prave kvuli tomu? BTW `state.tlog` ma i pro jednoho robota 3.7GB, rozbaleny)

**Comment by zwn** [comment]
Ale stejně mi přijde skoro neuvěřitelný, kolik toho jeden robot dokáže projet... a neztratit se.

**Comment by tajgr** [comment]
Náhodou jsem byl u toho, když Freyja dojížděla domů a získala 5 bodů. Pár pozorování:
- nefungovalo spojení skrze breadcrumbs. Freyja musela dojet až skoro na start.
- Do výklenků jen nakukuje, vlastně se tam ani pořádně nenatočí. Pokud by tam byl menší artefakt, nemusela by ho vidět (bez boční kamery). Telefon by byl bez šance.
- Některé vzdálené chodby končí propadlem/šachtou, ve kterých je schovaný artefakt (min dva případy). Tj. Drona tam samotná nedoletí a pozemní robot se tam nedostane. Jako kdyby to bylo připraveno pro "vačnatce".

Jinak ale krásná jízda a krásných 5 bodů!

**Comment by jisa** [comment]
> * Do výklenků jen nakukuje, vlastně se tam ani pořádně nenatočí. Pokud by tam byl menší artefakt, nemusela by ho vidět (bez boční kamery). Telefon by byl bez šance.

Ano, tak jak ted jezdi, tam muze sedet survivor a stejne ho neuvidime. Bocni detekce budou potreba. Ale to je dalsi CPU v haji ... :-/

> * Některé vzdálené chodby končí propadlem/šachtou, ve kterých je schovaný artefakt (min dva případy). Tj. Drona tam samotná nedoletí a pozemní robot se tam nedostane. Jako kdyby to bylo připraveno pro "vačnatce".

Ono i napriklad ve FP2 hned na zacatku sedi survivor uprostred schodiste a o kousek dal je podobne schovany batoh.

My s temi dronami ale (zatim?) letame dost pomalu. Rychle letici drona by mohla mit podobny dolet, jako tahle taky pomerne pomalu jedouci Freyja dojezd. Obecne bych rekl, ze v Tunnel a Urban, kde jedeme/letime rovnym volnym prostorem, muzeme zrychlit.
